### PR TITLE
Disable pthread for standalone wasm build support

### PR DESCRIPTION
**Pull request by Shikugawa**
Introduced wasm support by adding knobs to disable for standalone wasm build.

### CODE REVIEW

File: absl/BUILD.bazel [hunk 1]
```diff
@@ -58,3 +58,8 @@ config_setting(
     },
     visibility = [":__subpackages__"],
 )
+
+config_setting(
+    name = "wasm",
+    values = {"cpu": "wasm"},
```
Comment (lizan) [inline absl/BUILD.bazel]: Let's use `wasm32` https://github.com/bazelbuild/platforms/blob/master/cpu/BUILD#L78

**Comment by Shikugawa** [comment]
@mbxx How is this?

**Comment by mbxx** [comment]
Oh, wow, I'm so sorry that this slipped my radar!  Thank you for pinging me.

This looks good to me, I just need to make sure this will import cleanly.

**Comment by rogeeff** [comment]
Mark, I'm working to merge this PR. Let me know if you want to do it.

**Comment by Hmdalali7** [comment]
Open

**Comment by Hmdalali7** [comment]
Open

**Comment by mbxx** [comment]
@rogeeff : Please, go ahead!  I don't need to be the one to merge this, I just want to make sure it doesn't get dropped again :)

**Pull request merged**